Jenkinsfile_CNP

```diff
@@ -103,7 +104,8 @@ env.DEFAULT_COLLECTION_ASSERTION_MODE="UNORDERED"
 env.MAX_NUM_PARALLEL_THREADS=6
 env.BEFTA_S2S_CLIENT_ID = "ccd_gw"
 env.ROLE_ASSIGNMENT_API_GATEWAY_S2S_CLIENT_ID = "ccd_data"
-env.BEFTA_TEST_STUB_SERVICE_BASE_URL = "http://ccd-test-stubs-service-aat.service.core-compute-aat.internal"
+env.BEFTA_TEST_STUB_SERVICE_BASE_URL = "https://ccd-test-stubs-service-pr-177.service.core-compute-preview.internal"
+env.TEST_STUB_SERVICE_BASE_URL = "http://ccd-test-stubs-service-pr-177-java"
```

Comment:
Would this not be the same value as above ?

Reply:
This is to be reverted before merging.